docs/user-guide/mvd-3270.md

```diff
@@ -106,7 +106,7 @@ Key sequences allow users to define their own key sequences. Each key sequence i
 * There is a set of predefined key sequences
 * Key sequences are accesible via top right menu:
   
-![image](https://github.com/Martin-Zeithaml/docs-site/assets/66114686/0052f8d2-92d3-4313-9087-65ab53ea41a3)
+![image](https://github.com/user-attachments/assets/bded3244-5071-4f21-8a13-b1a952aa1e0d)
```

Comment:
Is there a reason this cannot be in the image file in the docs-site repo?
Under docs, there is a file called images.